### PR TITLE
Fix notebook and getting started examples [skip ci]

### LINKED ISSUE #6416: [BUG] Example Jupyter notebook fails to parse and contains errors
**Describe the bug**
https://github.com/NVIDIA/spark-rapids/blob/branch-22.08/docs/demo/GCP/Mortgage-ETL.ipynb is not loadable in dataproc. Copying over the code also fails with errors due to resource limitation.

**Steps/Code to reproduce bug**
Attempt to load the notebook into a dataproc cluster. Copying each code cell and execution will fail with errors in example T4 cluster. A100 cluster fails to launch with example configuration.

**Additional context**
22.08 release

### PR DESCRIPTION
**Pull request by rwlee**
Resolves #6416

Getting started rewording, fix the notebook (running errors and un-parseable by dataproc jupyter instances), link fixes, and example script fixes.

The notebook for 22.08 training still does not work because it needs `spark.dynamicAllocation.enabled=false`. I have not had time to look into how to fix this issue.

### CODE REVIEW

**Comment by sameerz** [comment]
@rwlee this is too late to get into 22.08.  Please retarget to 22.10.

File: docs/get-started/getting-started-gcp.md [hunk 1]
```diff
@@ -143,34 +145,18 @@ If you'd like to further accelerate init time to 4-5 minutes, create a custom Da
...
-Precompiled
-[XGBoost4j](https://repo1.maven.org/maven2/ml/dmlc/xgboost4j-gpu_2.12/) and [XGBoost4j
-Spark](https://repo1.maven.org/maven2/ml/dmlc/xgboost4j-gpu_2.12/) libraries can be
-downloaded from maven.  They are pre-downloaded by the GCP [RAPIDS init
-action](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/rapids).
+The sample notebook will transcode the CSV files into Parquet files before running an ETL query that prepares
+the dataset for training.  The ETL query splits the data, saving 20% of the data in a seaprate GCS location training for evaluation.  Using the default notebook configuration the first stage should take ~110 seconds (1/3 of CPU execution time with same config) and the second stage takes ~170 seconds (1/7 of CPU execution time with same config).  The notebook depends on the pre-compiled [Spark RAPIDS SQL plugin](https://mvnrepository.com/artifact/com.nvidia/rapids-4-spark) which is pre-downloaded by the GCP Dataproc [RAPIDS init script](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/rapids).
```
Comment (viadea) [inline docs/get-started/getting-started-gcp.md:155]: @rwlee in doc, each line should be < 100 characters. Could you split the long line into many small lines? 

Same for some other long times.

**Review by viadea — CHANGES_REQUESTED** [review]

**Comment by nvliyuan** [comment]
LGTM

**Review by nvliyuan — APPROVED** [review]
LGTM

**Review by viadea — DISMISSED** [review]

**Comment by rwlee** [comment]
build

**Comment by rwlee** [comment]
build

**Review by nvliyuan — APPROVED** [review]

**Comment by rwlee** [comment]
build

**Comment by rwlee** [comment]
build

**Comment by jlowe** [comment]
CI failed due to #6515.  Skipping CI since this is just changes to documentation.

**Comment by jlowe** [comment]
build

**Pull request merged**